src/pool/srv_pool.c

```diff
@@ -6500,6 +6500,45 @@ pool_svc_schedule_reconf(struct pool_svc *svc, struct pool_map *map, uint32_t ma
 	return 0;
 }
 
+static void
+pool_map_crit_prompt(struct pool_svc *svc, struct pool_map *map, d_rank_t rank)
```

Comment:
Shouldn't a RAS event being raised? And I think the ras event function will print local error logs accordingly.

Reply:
TBH I am not sure if current handling for pw_rf will be changed in 2.6.2. We will simply the container RF/rd_lvl things later. maybe prompt with critical log is fine for 2.6.1's temporarily solution, thx.